tests/wpt/web-platform-tests/dom/nodes/Node-cloneNode.html

```diff
@@ -27,25 +27,246 @@
   }
 }
 
+function check_copy(orig, copy, type) {
+    assert_not_equals(orig, copy);
+    assert_equal_node(orig, copy);
```

Comment:
This function `check_copy` now utilizes the `assert_equal_node` function earlier in this file